pmlc/dialect/tile/lowering.cc

```diff
@@ -625,7 +646,7 @@ OwningModuleRef LowerIntoStripe(ModuleOp workspace) {
     IVLOG(1, "LowerIntoStripe failed: " << mlir::debugString(*module->getOperation()));
     throw std::runtime_error("Lowering to stripe dialect failure");
   }
-  // IVLOG(1, "after:\n" << mlir::debugString(*module->getOperation()));
+  IVLOG(1, "after:\n" << mlir::debugString(*module->getOperation()));
```

Comment:
This seems like a slightly verbose logging level.

Reply:
Oops, meant to re-comment or remove it